sound/hybrid_font.h

```diff
@@ -548,7 +548,20 @@ class HybridFont : public SaberBase {
     return current_effect_length_;
   }
 
-
+  void SB_LowBatt() override
+  {
+    //play the fonts low battery sound if it exists
+    //TODO: if lowbatt does not exist in the font, then play a global sound (instead of a beep or a talkie)
+    if (SFX_lowbatt) {
+      PlayCommon(&SFX_lowbatt);
+    } else {
+      STDOUT << "Battery low :" << battery_monitor.battery() << "\n";
```

Comment:
This printout is not needed since already have something very similar in CheckLowBattery()